hw/ip/otp_ctrl/data/otp_ctrl_mmap.hjson

```diff
@@ -184,6 +184,46 @@
                     name: "CREATOR_SW_CFG_RMA_SPIN_CYCLES",
                     size: "4"
                 }
+                {
+                    name: "CREATOR_SW_RNG_REPCNT_THRESHOLDS",
+                    size: "4"
+                }
+                {
+                    name: "CREATOR_SW_RNG_REPCNTS_THRESHOLDS",
```

Comment:
nit: I'm not very familiar with entropy_src but just wanted to confirm: Is the plurality of REPCNT the only difference between this item and the one above?

Reply:
That is correct. `REPCNTS` is a different configuration compared to `REPCNT`. Single bit versus symbol based repetition count tests. I believe symbol bases does it in an N-bit granularity based on how many RNG bitstreams are provided by AST.

Reply:
Thanks for the explanation @moidx!